compose/config/fields_schema.json

```diff
@@ -37,7 +37,15 @@
 
         "environment": {
```

Comment:
I honestly can't remember now why I thought they had separate requirements and I can't find it in my notes. If you can't think of any issues with doing that, I think that's a good suggestion, I will amend. Thanks.